tools/standardize_config_json.py

```diff
@@ -1,17 +1,16 @@
-"""
-A simple script to ensure that the ordering of the "service" section in config.json is standardized.
-NOTE: This overwrites the files.
+#!/usr/bin/env python3
 
-Usage: from the `dcos-commons` root:
-
-$ python tools/update_config_json.py
-
-"""
 import collections
+import difflib
 import json
 import logging
+import os.path
 import sys
-import difflib
+
+usage = """Standardizes the ordering of sections in config.json files.
+
+Usage:
+$ ./standardize_config_json.py $path_to_config_json $path_to_dcos_commons_json"""
```

Comment:
`...dcos_commons_json` does not sound like a good name, because it does not convey the intent. `...formatting_policy_json` perhaps?

Reply:
My idea with this file was having configuration for any dcos-commons tool (i.e. a "dcos-commons tooling" configuration file). If you notice the example configuration, there's a key specific for this script's configuration (`"standardize_config_json"`). If more scripts need to be configured in a way that doesn't lend itself well to command line arguments (like it's the case here) they could have their own sections.

Maybe we could rephrase the variable name in the usage output (and in the script itself) so that it's intent is clearer. Do you have suggestions?

Reply:
Oh, I see. Perhaps we should have a config file per tool, then? This seems to be the pattern for most UNIX-like utilities, rather than having "the config file" with sections per tool...

Reply:
We could. What I had in mind was less UNIX-like and more [`pyproject.toml`](https://black.readthedocs.io/en/stable/pyproject_toml.html)-like, so as to avoid requiring services that use dcos-commons tooling as a _unit of dependency_ to have to have in their source control several configuration files.

Reply:
:thinking: that makes sense if was not for the naming... `pyproject` is much more convenient than `dcos-commons-tools-consuming-project` :-/

Reply:
`sdktools.json`?

Reply:
Sounds good to me 👍

Reply:
Fixed in 072eb00997b0f6d39c184bcd8501dc19f34db4e9 and https://github.com/mesosphere/dcos-elastic-service/pull/21/commits/bfc8a75c3178176a132e67b3ea3f8679ea8d0e99.